### PR TITLE
Add nightly recipes

### PR DESCRIPTION
**Pull request by matthiasclasen**

### CODE REVIEW

File: source/apps.html.haml [hunk 1]
```diff
@@ -371,6 +371,13 @@ description: Applications distributed as Flatpaks, ready to download.
                 <span class="unselectable">$ </span>flatpak install --from https://git.gnome.org/browse/gnome-apps-nightly/plain/polari.flatpakref
                 <span class="unselectable">$ </span>flatpak run org.gnome.Polari
 
+            %a{:name => "recipes-unstable"}
+            %h5 Polari
```
Comment (fmuellner) [inline source/apps.html.haml]: Recipes :-)

**Comment by AsciiWolf** [comment]
I think it could be a good idea to also add the unstable Recipes to "Click to Download" list, since other GNOME unstable apps are there too.

**Comment by matthiasclasen** [comment]
You know what ? This apps page is too hard to update.

**Comment by jimmac** [comment]
Matthias, when you say it's hard to update, you mean the stable/unstable split? Should the helper just take a name, description and two URLs for stable/nightly?

**Comment by AsciiWolf** [comment]
I see there's still an old url (matthiasclasen.github.io) for stable Recipes in the Command Line instructions.

**Comment by matthiasclasen** [comment]
^ thats what I mean: 4 places to update per app, and in the end its still not right.

**Comment by jimmac** [comment]
Pushed a restructured app list page. Now the apps are maintained in the data/apps.yml file

**Pull request merged**